plugins/plugins64.xml

```diff
@@ -1099,4 +1099,23 @@
 			</x64>
 		</remove>
 	</plugin>
+	<plugin name="npp.Connections">
```

Comment:
I will move this section up so diff with https://github.com/bruderstein/npp-plugins-x64/blob/master/plugins/plugins_template.xml is easily possible.

Reply:
Ok